### DISCUSSION TITLE
Whatsapp Notifactions (Popup- NO sound) for single contacts?

### DISCUSSION THREAD

**Discussion by Cozynezz** [ferdium/ferdium-app#1594] [General]
Hey guys, it is possible, to receive pop message notifications only for specific contacts in my friendlist?

**Comment by vraravam** [comment]
Ferdium is only showing the `http://web.whatsapp.com` site when speaking about whatsapp. Nothing fancy other than that.

Consequently, if such a feature (more personalized notifications) is supported by the web ui of the said service/site, Ferdium will automatically support it. So, in effect, the answer to your question is "No".

**Reply by Cozynezz** [reply]
okay thank you!
